VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -69,9 +76,54 @@ public boolean onQueryTextChange(String newText) {
                 return false;
             }
         });
+        searchView.setOnQueryTextFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean queryTextFocused) {
+                if (!queryTextFocused) {
+                    searchActive = false;
```

Comment:
Can you not just check for `searchView.hasFocus()`, instead of having to set this flag?